imports/client/components/Avatar.tsx

```diff
@@ -99,13 +105,15 @@ const Avatar = React.memo(
     displayName,
     discordAccount,
     className,
+    isSelf,
```

Comment:
I think that `$isSelf` either needs to be optional on `AvatarContainer` or this needs to default to false, a la

```suggestion
    isSelf = false,
```

Mild preference for the latter I think?